index.css

```diff
@@ -607,8 +607,266 @@ a:focus {
     /*becuase desktop nav use sticky so change to auto is ok every pages*/
     margin-top: auto;
   }
-  .main-aboutUs .members {
+  .main-aboutUs, .members {
     display: flex;
   }
 }
 /* activities-events */
+
```

Comment:
before here we already did our screen sizes so you have to put all your css before that